t/Makefile.am

```diff
@@ -482,6 +482,7 @@ check_LTLIBRARIES = \
 	job-manager/plugins/create-reject.la \
 	job-manager/plugins/perilog-test.la \
 	job-manager/plugins/config.la \
+	job-manager/plugins/jobspec-update.la \
```

Comment:
just double checking, is last sentence of commit message finished?  looks like maybe didn't finish sentence?

also typo, "jobspec-updte" -> "jobspec->update"

Reply:
Good catch. I meant to drop that sentence fragment.